contracts/MerklePatriciaProofMock.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.4.23;
+
+import "./MerklePatriciaProof.sol";
+import "./util.sol";
+
+/**
+ *	@title This contract is an mock contract for MerklePatriciaProof library contract.
+ *
+ *	@notice It passes the parameters which it receives from test cases.
+ *
+ */
+contract MerklePatriciaProofMock is Util{
+
+    function MerklePatriciaProofMock(){
+
+    }
+
+    /**
+     * @dev This method is used for Account proof
```

Comment:
@notice is missing for the function

cleaner will be if we have new line between @dev and @param to group commenting